scripts/obtain/dotnet-install.sh

```diff
@@ -646,7 +650,7 @@ downloadwget() {
     if [ -z "$out_path" ]; then
         wget -q --tries 10 -O - "$remote_path" || failed=true
     else
-        wget -v --tries 10 -O "$out_path" "$remote_path" || failed=true
+        wget --tries 10 -O "$out_path" "$remote_path" || failed=true
```

Comment:
Why are we changing the wget call?

Reply:
Alpine's wget doesn't accept the -v flag.

Reply:
Btw, the `-v` sets a verbose mode that's on by default anyways.